webapp/src/Controller/API/ContestController.php

```diff
@@ -263,6 +263,113 @@ public function setBannerAction(Request $request, string $cid, ValidatorInterfac
         return new Response('', Response::HTTP_NO_CONTENT);
     }
 
+    /**
+     * Delete the text for the given contest.
+     */
+    #[IsGranted('ROLE_ADMIN')]
+    #[Rest\Delete('/{cid}/text', name: 'delete_contest_text')]
+    #[OA\Response(response: 204, description: 'Deleting text succeeded')]
+    #[OA\Parameter(ref: '#/components/parameters/cid')]
+    public function deleteTextAction(Request $request, string $cid): Response
+    {
+        $contest = $this->getContestAndCheckIfLocked($request, $cid);
+        $contest->setClearContestText(true);
+        $contest->processContestText();
+        $this->em->flush();
+
+        $this->eventLogService->log('contests', $contest->getCid(), EventLogService::ACTION_UPDATE,
+            $contest->getCid());
+
+        return new Response('', Response::HTTP_NO_CONTENT);
+    }
+
+    /**
+     * Set the text for the given contest.
+     */
```

Comment:
```suggestion
```

Reply:
Fixing this later in a consistency commit